util.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var util = module.exports = require('util');
```

Comment:
this will require util in the build. Please don't do this. Just use the functions here to avoid bringing in all of util when packaged.